### PR TITLE
home and sign in sign up

### PR DESCRIPTION
**Pull request by dev-ashishsethi**

### CODE REVIEW

File: assets/images/R&B _ SignIn.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>R&amp;B | SignIn</title>
+    <link rel="icon" href="http://127.0.0.1:5500/assets/images/RB-icon.png" type="image/icon type">
+    <link rel="stylesheet" href="./R&amp;B _ SignIn_files/style.css">
+    <link rel="stylesheet" href="./R&amp;B _ SignIn_files/all.min.css">
+</head>
+<body>
+    <nav class="nav-container">
+        <a href="http://127.0.0.1:5500/signIn%20and%20SignUp/index.html">
```
Comment (eldhoabraham94) [inline assets/images/R&B _ SignIn.html:14]: check hard coded address

File: assets/images/R&B _ SignIn.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            <i class="fas fa-heart navbar-icons"></i>
+            <i class="fas fa-shopping-cart navbar-icons"></i>
+            <button class="btn btn-primary">Log In</button>
+        </div> 
+    </nav>
+
+    <div class="form-container">
+        
+            <!-- <h1 style="color: aqua;"class="form-subcontainer">LOL</h1> -->
+        <form action="http://127.0.0.1:5500/signIn%20and%20SignUp/signIn.html" class="signIn-form">
```
Comment (eldhoabraham94) [inline assets/images/R&B _ SignIn.html:34]: local links

File: style.css [hunk 3]
```diff
@@ -0,0 +1,195 @@
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:ital,wght@0,100;0,200;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&family=Nunito&family=Roboto:wght@100;300;400;500;700&display=swap');
+@import url('./signIn and SignUp/signin.css');
+@import url('./signIn and SignUp/signUp.css');
+:root{
+    --color-black:rgb(20 20 20);
```
Comment (pnchinmay) [inline style.css:5]: keep the format same, either hex or rgb

**Review by pnchinmay — APPROVED** [review]

**Review by eldhoabraham94 — APPROVED** [review]
Good to go

File: assets/images/R&B _ SignIn_files/style.css [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+a:visited{
+    color: var(--color-white);
+}
+
+.input-box{
+    align-self: center;
+    display: inline-flex;
+    background-color: transparent;
+    border-radius: 5px;
+    padding: 6px;
```
Comment (dipen2000) [inline assets/images/R&B _ SignIn_files/style.css:63]: use rems here

File: assets/images/R&B _ SignIn_files/style.css [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+    flex-direction: column;
+    color: var(--color-white);
+    background-color: var(--color-black);
+    width: 100%;
+    height: 4rem;
+}
+
+.text{
+    background-color: var(--color-black);
+    margin-bottom: 7px;
```
Comment (dipen2000) [inline assets/images/R&B _ SignIn_files/style.css:173]: have rems

File: signIn and SignUp/signIn.html [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+            </label>
+            <input type="text" placeholder="Enter Email" class="sign-input-box" id="email-input">
+
+            <label for="pass-input" class="form-label" id="pass-label">
+                Password
+            </label>
+            <input type="password" placeholder="Enter Email" class="sign-input-box" id="pass-input">
+            
+            
+
```
Comment (dipen2000) [inline signIn and SignUp/signIn.html:51]: extra white spaces

File: signIn and SignUp/signin.css [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+    grid-area: pass;
+}
+
+#form-btn1{
+    grid-area: btn1;
+}
+#form-btn2{
+    grid-area: btn2;
+    text-align: center;
+    font-size: 13.33px;
```
Comment (dipen2000) [inline signIn and SignUp/signin.css:38]: rem

**Review by dipen2000 — APPROVED** [review]
Looks good

**Pull request merged**